v2/internal/testcommon/vcr/v3/tracking_roundtripper.go

```diff
@@ -34,17 +34,19 @@ const (
 type requestCounter struct {
 	inner http.RoundTripper
 
-	countsMutex sync.Mutex
-	counts      map[string]uint32
-	azureIDs    creds.AzureIDs
+	countsMutex     sync.Mutex
+	counts          map[string]uint32
+	azureIDs        creds.AzureIDs
+	customRedactMap map[string]string
```

Comment:
and here

Reply:
I've replaced the map with a redactor struct and renamed wherever possible.